test/e2e/data/infrastructure-aws/kustomize_sources/limit-az/patches/elb-health-check-protocol.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
```

Comment:
Is there any reason to put it inside `limit-az` pkg?

Reply:
no reason, it was suggested by @sedefsavas that we take any of the existing test case to check for upgrade case for this optional field.